dlpy/transformers/bert_model.py

```diff
@@ -704,15 +837,15 @@ def _write_huggingface_bert_hdf5(self, sas_layer_info, task_parms):
                             g_out.create_dataset(dset_name, data=task_parms[lname]['weights'])
 
                             # save bias in format amenable to SAS
-                            if vecval is not None:
+                            if 'bias' in task_parms[lname].keys():
```

Comment:
You don't need .keys() here.  'bias' in task_params[lname] is sufficient.

Reply:
I fixed the requested issue and other similar issues in bert_model.py and bert_utils.py.  I also updated the unit tests that covered this code and reran them to ensure I didn't introduce any errors.